packages/pwa/README.md

```diff
@@ -29,6 +29,26 @@ Now that the development server is running, you can open a browser and preview y
 
 See the [Localization README.md](./app/translations/README.md) for important setup instructions for localization.
 
+## Url Customisation
+
+The Retail React App allows you to customise your locale for your app urls.
+It can be either set as `path` or `query_param` or `none`.
```

Comment:
```suggestion
The Retail React App allows you to customize certain aspects of the url. Currently you can choose where the current locale is displayed, whether it is in the urls path, query parameters, or not displayed. Below are the details of the possible configuration values.
```

Reply:
I re-worded things a little bit. Specially removing the use of the actual values since you are listing them below.